src/App.tsx

```diff
@@ -5,6 +5,40 @@ import { customersGetter, customersSetter } from './api/CustomerApi';
 import { initialCustomers } from './api/startingValues';
 import ContainerWrapper from './components/ContainerWrapper';
 import Landing from './pages/Landing';
+import { QueryClient, QueryClientProvider, useQueryClient, useMutation } from 'react-query';
+import App from './App';
+import ReactDOM from 'react-dom';
+
+const queryClient = new QueryClient();
+
+ReactDOM.render(
+  <QueryClientProvider client={queryClient}>
+    <App />
+  </QueryClientProvider>,
+  document.getElementById('root')
+);
+
+const addNewCustomer = async (customerData: any) => {
+  const response = await fetch('/api/customers', {
+    method: 'POST',
+    body: JSON.stringify(customerData),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  if (!response.ok) {
+    throw new Error('Failed to add customer');
+  }
+
+  return response.json();
+};
```

Comment:
This is a standard api call, but to an api that does not exist. 
If you read task 3, it tells you that you have to use the api (that is already completed for you)[here](https://github.com/content-services/React-typescript-demo/blob/main/src/api/CustomerApi.ts#L38).